### PR TITLE
DevOps: move project containers to github package using git workflow

### PR DESCRIPTION
**Pull request by omertalmi5**
According to the last news from the docker hub we moving all containers of all projects to store the project's images on GitHub, using github packages.

This PR change and refactor the git workflow CI that already exists.

```
change: main file - push image to GitHub package.
```

I checked only the building and pushing of the image.
**I don't know how to check that the deploying is correct.**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,41 +1,36 @@
...
   deploy:
-    #if: github.repository == 'hasadna/anyway' && github.ref == 'refs/heads/master'
     needs:
       - Build-anyway-reports
     runs-on: ubuntu-18.04
     steps:
     - uses: actions/checkout@v2
     - env:
-        DOCKER_REPOSITORY_ANYWAY_REPORTS: ${{ env.DOCKER_REPOSITORY_ANYWAY_REPORTS }}
+        DOCKER_REPOSITORY_ANYWAY_REPORTS: ${{ env.SERVER }}/${{ env.DOCKER_REPOSITORY_ANYWAY_REPORTS }}
```
Comment (OriHoch) [inline .github/workflows/main.yml:33]: @omertalmi5 why is this change needed? what's the content of env.SERVER?
I think that the deploy script should not change as it only needs the docker image which is available in env.DOCKER_REPOSITORY_ANYWAY_REPORTS

**Review by OriHoch — CHANGES_REQUESTED** [review]
looks great, thanks!
regarding deploy there shouldn't be any change, it should work as-is

see minor change, also we can't merge yet as I need to do some change on hasadna-k8s to support pulling images from github packages, I'll try to do it this week and then we can merge

**Comment by OriHoch** [comment]
I made the changes on the cluster to support pulling these packages, so once the requested change is done, you can merge

**Comment by omertalmi5** [comment]
> I made the changes on the cluster to support pulling these packages, so once the requested change is done, you can merge



> I made the changes on the cluster to support pulling these packages, so once the requested change is done, you can merge

Do I need to add the registry name in the begining of the image name, for the k8s would know fron where to pull?

**Review by OriHoch — APPROVED** [review]
you're right, need to specify the registry server, looks good now, @atalyaalon  ready for merge

**Pull request merged**